File: Try_GameEngine5/src/com/example/try_gameengine/viewport/ImageInfo.java
package com.example.try_gameengine.viewport;

public class ImageInfo {
	int x = 0;
	int y = 0;
	int w = 0;
	int h = 0;
	int layer = 0;
	
	public ImageInfo(){
		
	}
	
	public ImageInfo(int x, int y, int w, int h, int layer){
		this.x = x;
		this.y = y;
		this.w = w;
		this.h = h;
		this.layer = layer;
	}

	public int getX() {
		return x;
	}

	public void setX(int x) {
		this.x = x;
	}

	public int getY() {
		return y;
	}

	public void setY(int y) {
		this.y = y;
	}

	public int getW() {
		return w;
	}

	public void setW(int w) {
		this.w = w;
	}

	public int getH() {
		return h;
	}

	public void setH(int h) {
		this.h = h;
	}

	public int getLayer() {
		return layer;
	}

	public void setLayer(int layer) {
		this.layer = layer;
	}
	
	
}

File: Try_GameEngine5/src/com/example/try_gameengine/viewport/ImageInfo.kt
package com.example.try_gameengine.viewport

class ImageInfo {
    var x: Int = 0
    var y: Int = 0
    var w: Int = 0
    var h: Int = 0
    var layer: Int = 0

    constructor()

    constructor(x: Int, y: Int, w: Int, h: Int, layer: Int) {
        this.x = x
        this.y = y
        this.w = w
        this.h = h
        this.layer = layer
    }
}
